### PR TITLE
[Workflows] Add supervision start date tooltip for ET

### LINKED ISSUE #2243: [Workflows] Add tooltip for supervision start date in ND
Add a hover tooltip for supervision start date in ND that says:
"This date is included in the filled form as the date that the Defendant was sentenced; in rare cases, the sentencing date and supervision start date may differ. Double check the date in the form to ensure that the sentencing date and supervision start date match in this case."

### PR DESCRIPTION
**Pull request by jovergaag**
## Description of the change

Adds a tooltip to the supervision start date details section for ND Early Termination client profile.

It also changes the info tooltips to link (on click) to the ND ET Methodology.

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [x] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] Configuration change (adjusts configuration to achieve some end related to functionality, development, performance, or security)

## Related issues

> Closes #2243 

### Development

These boxes should be checked by the submitter prior to merging:

- [x] Manual testing against realistic data has been performed locally
- [ ] E2E have been run for Lantern changes ([Steps in the Readme](https://github.com/Recidiviz/pulse-dashboard#running-e2e-tests))

### Code review

These boxes should be checked by reviewers prior to merging:

- [ ] This pull request has a descriptive title and information useful to a reviewer
- [ ] This pull request has been moved out of a Draft state, has no "Work In Progress" label, and has assigned reviewers
- [ ] Potential security implications or infrastructural changes have been considered, if relevant

### CODE REVIEW

File: src/WorkflowsStore/Client.ts [hunk 1]
```diff
@@ -86,6 +86,21 @@ const SUPERVISION_LEVEL_MAP: Record<string, SupervisionLevel> = {
   "STANDARD: MINIMUM": "Minimum",
 };
 
+type ClientDetailsCopy = {
+  supervisionStartDate?: {
+    tooltip: string;
+  };
+};
+
+export const CLIENT_DETAILS_COPY: Record<string, ClientDetailsCopy> = {
```
Comment (jovergaag) [inline src/WorkflowsStore/Client.ts:95]: I figured that we may start having custom tooltip copy for the different client details per state, so building out this structure here such that it can eventually be replaced by the content system synced from google sheets is the the most future proof implementation.

File: src/core/utils/constants.ts [hunk 2]
```diff
@@ -74,9 +71,15 @@ export const supervisionFiltersOrder = [
...
   US_TN:
     "https://drive.google.com/file/d/1fkqncNb_GNYBvRfOgij4QHw4HEdkkHHz/view?usp=sharing",
   US_ND:
     "https://drive.google.com/file/d/1eHbSEOjjT9FvxffSbXOYEfOYPJheeu6t/view",
 };
+
+export const WORKFLOWS_POLICIES_URL: Record<string, string> = {
+  US_TN:
+    "https://drive.google.com/file/d/1YNAUTViqg_Pgt15KsZPUiNG11Dh2TTiB/view?usp=sharing",
+  US_ND: WORKFLOWS_METHODOLOGY_URL.US_ND,
```
Comment (jovergaag) [inline src/core/utils/constants.ts:84]: ND does not have a policy document to link to, the product team asked to link to the methodology instead.

File: src/core/utils/constants.ts [hunk 3]
```diff
@@ -74,9 +71,15 @@ export const supervisionFiltersOrder = [
   FILTER_TYPES.RACE,
 ];
 
-export const WORKFLOWS_METHODOLOGY_URL: Partial<Record<TenantId, string>> = {
+export const WORKFLOWS_METHODOLOGY_URL: Record<string, string> = {
```
Comment (macfarlandian) [inline src/core/utils/constants.ts:74]: why this change? I think a narrower type is better for object keys (same applies to the new variable exported below)

File: src/core/utils/constants.ts (hunk 2 above)
Comment (macfarlandian) [reply src/core/utils/constants.ts:84]: in that case maybe rename this to reflect its intended use instead?

**Review by macfarlandian — APPROVED** [review]
looks good but a few nits

File: src/WorkflowsStore/Client.ts [hunk 4]
```diff
@@ -86,6 +86,21 @@ const SUPERVISION_LEVEL_MAP: Record<string, SupervisionLevel> = {
   "STANDARD: MINIMUM": "Minimum",
 };
 
+type ClientDetailsCopy = {
+  supervisionStartDate?: {
+    tooltip: string;
```
[the remark this answers is not in the document]
Comment (jovergaag) [reply src/WorkflowsStore/Client.ts:91]: At this point the only copy that I can imagine we will add is the tooltip to explain the various client attributes. So I made it required. I think this will all change significantly once we get the content sync system up and running so I didn't focus too much on the types.

File: src/core/utils/constants.ts (hunk 3 above)
Comment (jovergaag) [reply src/core/utils/constants.ts:74]: The `TenantId` typing is a pretty big mess. I tried to make this work keeping the type but there are places now where the `client.stateCode` is used to access these objects, and it felt even worse to type cast those from `string` to `TenantId`.

I'm thinking that once ND is done we could try to do a tenant typing refactor to try to separate out workflows tenant as their own type, which would make things like this cleaner.

Comment (macfarlandian) [reply src/core/utils/constants.ts:74]: 😭 yeah that sounds like the right approach
